index.src.html

```diff
@@ -4492,9 +4504,12 @@
 
 <xmp class='idl'>
 enum VideoTransferCharacteristics {
-  "bt709",         // BT.709
-  "smpte170m",     // BT.601 (functionally the same as bt709)
-  "iec61966-2-1",  // sRGB
+  "bt709",
+  "smpte170m",
+  "iec61966-2-1",
+  "linear",
+  "pq",
+  "hlg",
```

Comment:
The alternative names here are "smpte2084" and "arib-std-b67", which would align with FFmpeg naming.

Reply:
Looks OK as is, but if you're considering changing, I'd suggest referencing BT2100, so something like `bt2100-2-pq` and `bt2100-2-hlg`.

Reply:
To clarify, preference is for `pq` and `hlg`

Reply:
Ah great :) I think Dan was just playing devils advocate above.

Reply:
I see! My comment was really: if you're going in that direction, prefer BT.2100 rather than the SMPTE and ARIB standards.